app/views/manager/areas/create.js.slim

```diff
@@ -0,0 +1,5 @@
+- if @area.errors.present?
+  | $("#message_errors").html("#{j(render 'shared/manager/error_messages', object: @area)}");
+    toastr.error("#{flash[:danger]}");
+- else
+  | location.reload();
```

Comment:
Em có thể dùng js để add thêm thông tin về area mới vào trang hiện tại được không?
Hoặc dùng `j render` để render lại một phần của view
Như thế sẽ nhanh và an toàn hơn là reload lại cả trang thế này

Reply:
em đã update lại rồi anh